examples/src/main/java/io/github/tcdl/examples/MultipleRequesterResponder.java

```diff
@@ -11,9 +11,12 @@
 /**
  * Created by anstr on 6/9/2015.
  */
-public class MultipleRequesterResponder extends BaseExample {
+public class MultipleRequesterResponder {
+
     private static final Integer NUMBER_OF_RESPONSES = 1;
 
+    private MSBUtil util = MSBUtil.getInstance();
```

Comment:
very strange method as for Util class - getInstance . AFAIS utility classes need not instance creation